yazi-core/src/tab/commands/jump.rs

```diff
@@ -33,17 +38,28 @@ impl Tab {
 		if opt.type_ == OptType::None {
 			return;
 		}
-
 		let cwd = self.current.cwd.clone();
+
 		tokio::spawn(async move {
 			let _guard = BLOCKER.acquire().await.unwrap();
 			let _defer = Defer::new(Scheduler::app_resume);
 			Scheduler::app_stop().await;
 
-			let result = if opt.type_ == OptType::Fzf {
-				external::fzf(FzfOpt { cwd }).await
-			} else {
-				external::zoxide(ZoxideOpt { cwd }).await
+			let result = match opt.type_ {
+				OptType::Fzf => external::fzf(FzfOpt { cwd }).await,
+				OptType::ZoxideInteractive => external::zoxide(ZoxideOpt { cwd, query: None }).await,
+				OptType::Zoxide => {
```

Comment:
I don't think the `anyhow` errors are necessary.

@sxyazi might have a cleaner way but I can get this feature to work by copying how the input is used elsewhere:

```rust
				OptType::Zoxide => {
					tokio::spawn(async move {
						let rx = Input::_show(InputCfg::zoxide());
						let rx = Debounce::new(UnboundedReceiverStream::new(rx), Duration::from_millis(50));
						pin!(rx);

						while let Some(Ok(s)) | Some(Err(InputError::Typed(s))) = rx.next().await {
							if let Ok(url) =
								external::zoxide(ZoxideOpt { cwd: cwd.clone(), query: Some(s) }).await
							{
								Tab::_cd(&url)
							}
						}
					});
					return;
				}
				_ => return,
			};
```

Also minor but `cargo +nightly fmt` or `rustfmt +nightly **/*.rs` to format the Rust files.